hi-ml-multimodal/src/health_multimodal/image/model/model.py

```diff
@@ -112,7 +125,9 @@ def train(self, mode: bool = True) -> Any:
             self.projector.train(mode=False)
         return self
 
-    def forward(self, x: torch.Tensor) -> ImageModelOutput:
+    def forward(self, *args: Any, **kwargs: Any) -> ImageModelOutput:
+        x = kwargs["x"]
+        assert isinstance(x, torch.Tensor)
```

Comment:
Why do we need these changes?

Reply:
[Discussed offline]

Reply:
Discussed offline - we don't, let's simplify :)